<?xml version="1.0"?>
<launch>
    <node name="RSM_PLC_connection" pkg="remodel_safety_manager" type="RSM_v1.py" output="screen">
    </node>
</launch>  
